public/components/agents/vuls/inventory/flyout.tsx

```diff
@@ -72,6 +72,25 @@ export class FlyoutDetail extends Component {
     const title = `${currentItem.cve}`;
     const id = title.replace(/ /g, '_');
 
+    const filterMap = {
+      name: 'data.vulnerability.package.name',
+      cve: 'data.vulnerability.cve',
+      architecture: 'data.vulnerability.package.architecture',
+      version: 'data.vulnerability.package.version',
+    };
+    const implicitFilters = [
+      { 'rule.groups': 'vulnerability-detector' },
+      { 'agent.id': this.props.agentId },
+      this.state.clusterFilter,
+      ...Object.keys(currentItem)
```

Comment:
It´s better to map 4 knonw fields than an uncertain number of fields

Reply:
Set the values to use the keys for `filterMap`'s which are known and constant instead of `currentItem`'s